packages/flutter_secure_storage/tizen/src/flutter_secure_storage.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter_secure_storage.h"
+
+#include <ckmc/ckmc-manager.h>
+#include <ckmc/ckmc-type.h>
+
+#include <algorithm>
+
+#include "log.h"
```

Comment:
`LOG()` is not used in this plugin. I think it's good to keep log.h for maintenance. But I don't think we need to add unused headers.